router/http/src/test/scala/io/buoyant/router/http/TracingFilterTest.scala

```diff
@@ -27,25 +26,28 @@ class TracingFilterTest extends FunSuite with Awaits {
     req.uri = "/foo?bar=bah"
     req.host = "monkeys"
     req.contentType = "text/plain"
+    req.contentLength = 94114
 
     Trace.letTracer(tracer) {
-      val f = service(req)
+      service(req)
```

Comment:
should this be wrapped in an `Await.result`?  Nothing in `service` is currently asynchronous, but this would turn flaky if that ever changes.